helm-chart/renku-core/templates/configmap.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "renku-core.fullname" . }}-nginx
+data:
+  nginx.conf: |
+    user nginx;
+    worker_processes 1;
+
+    events {
+      worker_connections 10000;
+    }
+
+    http {
+      {{- range $version := .Values.versions }}
+      upstream {{ $version.name }} {
+        server {{ include "renku-core.fullname" $ }}-{{ $version.name }};
+      }
+      {{ end }}
+
+      server {
+        listen 80;
+
+        server_name {{ include "renku-core.fullname" . }};
+
+        sendfile on;
+        tcp_nopush on;
+
+        location /renku/deployments {
+          root /;
+          add_header Content-Type application/json;
+          try_files /usr/share/nginx/html/deployments.json =404;
+        }
+
+        {{- range $version := .Values.versions }}
+        location /renku/{{ $version.prefix }} {
+          rewrite /renku/{{ $version.prefix }}/(.*) /renku/$1  break;
+          proxy_set_header Host $host;
+          proxy_pass http://{{ $version.name }};
+        }
+        {{- end }}
+
+        location /renku {
+          proxy_set_header Host $host;
+          proxy_pass http://v9;
```

Comment:
I think this should not be hardcoded. Either you can just use the first element in the versions array or add another chart value to specify the default version.